packages/vpnkit/vpnkit.0.1.0/opam

```diff
@@ -0,0 +1,71 @@
+opam-version: "1.2"
+maintainer: "David Scott <dave.scott@docker.com>"
+authors: [
+  "Anil Madhavapeddy <anil@recoil.org>"
+  "David Scott <dave.scott@docker.com>"
+  "David Sheets <dsheets@docker.com>"
+  "Gaetan de Villele <gdevillele@gmail.com>"
+  "Ian Campbell <ian.campbell@docker.com>"
+  "Magnus Skjegstad <magnus@skjegstad.com>"
+  "Mindy Preston <mindy.preston@docker.com>"
+  "Sebastiaan van Stijn <github@gone.nl>"
+  "Thomas Gazagnaire <thomas@gazagnaire.com>"
+  "Thomas Leonard <thomas.leonard@docker.com>"
+]
+homepage:     "https://github.com/moby/vpnkit"
+bug-reports:  "https://github.com/moby/vpnkit/issues"
+dev-repo:     "https://github.com/moby/vpnkit.git"
+doc:          "https://moby.github.io/vpnkit/"
+
+build: [
+  [make]
+]
+build-test: [
+  [make "test"]
+]
+install: [make "install" "BINDIR=%{bin}%"]
+remove: [make "uninstall" "BINDIR=%{bin}%"]
```

Comment:
these build instructions need to be ported to their jbuilder equivalents, I believe...